game/theater/iadsnetwork/iadsnetwork.py

```diff
@@ -118,34 +125,50 @@ def skynet_nodes(self, game: Game) -> list[SkynetNode]:
             if game.iads_considerate_culling(node.group.ground_object):
                 # Skip culled ground objects
                 continue
-            try:
-                skynet_node = SkynetNode.from_group(node.group)
-                for connection in node.connections.values():
-                    if connection.ground_object.is_friendly(
-                        skynet_node.player
-                    ) and not game.iads_considerate_culling(connection.ground_object):
-                        skynet_node.connections[connection.iads_role.value].append(
-                            SkynetNode.dcs_name_for_group(connection)
-                        )
-                skynet_nodes.append(skynet_node)
-            except IadsNetworkException:
-                # Node not skynet compatible
-                continue
+
+            skynet_node = SkynetNode.from_group(node.group)
+            for connection in node.connections.values():
+                is_friendly = connection.ground_object.is_friendly(skynet_node.player)
+                is_culled = game.iads_considerate_culling(connection.ground_object)
+                if is_friendly and not is_culled:
+                    key = connection.iads_role.value
+                    name = SkynetNode.dcs_name_for_group(connection)
+                    skynet_node.connections[key].append(name)
+            skynet_nodes.append(skynet_node)
+
```

Comment:
Simply dropping the entire try-except block, the only exception that could be triggered is in `dcs_name_for_group` on line 54, which imo is an exception we want to know about because if I understood the code, it's a case that shouldn't be possible given its current use.